src/Uno.UI/UI/Xaml/Controls/Popup/PopupPanel.cs

```diff
@@ -296,4 +296,10 @@ internal override bool IsViewHit()
 
 		return Popup?.IsLightDismissEnabled ?? false;
 	}
+
+	// This shouldn't be needed, but due to the way we implement Popups, we might
+	// need to make Popups not hit-testable even if the PopupPanel has a Background.
+#if __SKIA__
+	internal override bool HitTest(Point point) => IsViewHit();
+#endif
 }
```

Comment:
Not really feeling comfortable with this. I'd personally prefer if we still keep the HitTestability.Visible check in `SearchDownForTopMostElementAt`. Then at some point we can try to remove IsViewHit completely, at least on Skia. That's kinda my personal preference.

Reply:
Relying on Visual here is probably buggy due to bugs in popup implementation itself that will need to be their own separate work. In this case, `IsViewHit` can return false while the visual can actually hit test. This basically makes me more convinced into keeping the check in the VisualTreeHelper as done with #17973. My opinion here is that #17973 is enough for now. It's not worth the hassle around trying to force HitTest to be a stronger condition.

For now, the only place we know it's not stronger is PopupPanel, and this PR forces it to be strongly by basically just calling IsViewHit.

(btw, if we proceeded with this PR, it will be a better idea to `return IsViewHit() && base.HitTest(point)` I think)